lib/src/r_tree/node.dart

```diff
@@ -94,23 +87,21 @@ abstract class Node<E> implements RTreeContributor {
 
   /// Adds the rectangle containing [item] to this node's covered rectangle
   include(RTreeContributor item) {
-    _minimumBoundingRect = _minimumBoundingRect == null ? item.rect : rect.boundingBox(item.rect);
+    _minimumBoundingRect = _minimumBoundingRect == noMBR ? item.rect : rect.boundingBox(item.rect);
   }
 
   /// Recalculated the bounding rectangle of this node
   Rectangle updateBoundingRect() {
     if (children.isEmpty) {
-      _minimumBoundingRect = null;
-      return noMBR;
+      return _minimumBoundingRect = noMBR;
```

Comment:
I think the old way of updating the value then return the value is clearer, merging the two operations into a single line makes it read too similiar to returning a boolean value here, i.e., `_minimumBoundingRect == noMBR`.